### PR TITLE
Safari deprecated SPDY in iOS 12.2 & macOS 10.4.4

### PR DESCRIPTION
**Pull request by daytonlowell**
Source: https://webkit.org/blog/8569/removing-legacy-spdy-protocol-support/

### CODE REVIEW

**Comment by daytonlowell** [comment]
Wait to merge until iOS 12.2 and macOS Mojave 10.4.4 are released.

File: features-json/spdy.json [hunk 1]
```diff
@@ -329,7 +331,8 @@
   },
   "notes":"See also support for [HTTP2](https://caniuse.com/#feat=http2), successor of SPDY.\r\n\r\nChrome will be removing support in 2016 (in favor of HTTP2), other browsers likely to follow.",
   "notes_by_num":{
-    
+    "1":"Deprecated as of macOS Mojave 10.4.4. To be completly removed in a future version of macOS.",
```
Comment (krzysiek1507) [inline features-json/spdy.json]:
```suggestion
 "1":"Deprecated as of macOS Mojave 10.4.4. To be completely removed in a future version of macOS.",
```

File: features-json/spdy.json [hunk 2]
```diff
@@ -329,7 +331,8 @@
...
+    "2":"Deprecated as of iOS 12.2. To be completly removed in a future version of iOS."
```
Comment (krzysiek1507) [inline features-json/spdy.json]:
```suggestion
    "2":"Deprecated as of iOS 12.2. To be completely removed in a future version of iOS."
```

Comment (daytonlowell) [reply features-json/spdy.json]: Good catch. Thanks!

Comment (HM100) [reply features-json/spdy.json]: You need to commit the change to appear live @daytonlowell

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**